test.sh

```diff
@@ -97,12 +93,36 @@ function setup_envs {
   export TEST_ASSET_ETCD=$kb_root_dir/bin/etcd
 }
 
+function is_installed {
+  if [ command -v $1 &>/dev/null ]; then
+    return 0
+  fi
+  return 1
+}
+
+function fetch_go_tools {
+  header_text "Checking for golangci-lint"
+  if ! is_installed golangci-lint; then
+    GO111MODULE=on go get -d github.com/golangci/golangci-lint/cmd/golangci-lint@v1.15.0 
+  fi
+
+  DEP_LATEST=$(git describe --abbrev=0 --tags)
```

Comment:
??

Reply:
bleh, I swear I can write a simple bash script...This is supposed to be for installing the latest tagged version of dep.